composer.json

```diff
@@ -9,10 +9,14 @@
     {
       "type": "composer",
       "url": "https://packages.drupal.org/8"
+    },
+    {
+      "type": "composer",
+      "url": "https://asset-packagist.org"
     }
   ],
   "require-dev": {
-    "behat/behat": "^3.0",
+    "behat/behat": "3.3.1",
```

Comment:
Does this need to be pinned at this exact version? Otherwise it should probably remain semantic.